### PR TITLE
Disabled Node + Allways Opened

### PR DESCRIPTION
**Pull request by marcelfalliere**
I hope this one will do it for you :)

### CODE REVIEW

**Comment by riophae** [comment]
Really nice work! :D

File: docs/components/DisablingNodes.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      options: [ {
+        id: 'fruits',
+        label: 'Fruits',
+        children: [ {
+          id: 'apple',
+          label: 'Apple 🍎',
+        }, {
+          id: 'appletwo',
+          label: 'Apple Two 📺',
+          _disabled: true,
```
Comment (riophae) [inline docs/components/DisablingNodes.vue]: I think `disabled` without a `_` should be good here.

File: docs/partials/guides.pug [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
     Then you can pass an array of these nodes as the `options` prop.
   p.tip
-    :markdown-it(inline)
+    :markdown-it
```
Comment (riophae) [inline docs/partials/guides.pug]: image
I got this after removing the `(inline)`...

File: docs/components/DisablingNodes.vue (hunk 1 above)
Comment (marcelfalliere) [reply docs/components/DisablingNodes.vue]: I prefer the prefix `_` to distinguish between business attributes and components logic attribute. But writing this I'm seeing that `id` and `label` are components logic attribute so ... `disabled` is good ! 👍

File: docs/partials/guides.pug (hunk 2 above)
Comment (marcelfalliere) [reply docs/partials/guides.pug]: I add to remove this `(inline)` to  make the dev script pass.

**Comment by marcelfalliere** [comment]
Ready to merge ?

**Comment by riophae** [comment]
@marcelfalliere Yeah! I am back and going to merge now.

**Comment by riophae** [comment]
It's merged now. Thanks for your contribution! :D

**Comment by riophae** [comment]
There are still some things I want to do before a new version could be released. Don't hurry!

**Comment by marcelfalliere** [comment]
Cool !!! Awesome.

Quick question : how could I use this version with a `npm install` ? When I try to do that on my repo, the `dist` folder is empty. 

Thanks

**Comment by riophae** [comment]
I'm sorry, but I don't see what you mean...

**Comment by marcelfalliere** [comment]
Ok, I'll rephrase. I'm using `vue-treeselect` in my app. So `vue-treeselect` is declared as a dependency in `package.json`... I would like to `npm install` the version of `vue-treeselect` that includes this PR. Any idea on how to do that ?

**Comment by riophae** [comment]
You can only do that after a new version gets released. If you want to play with the version that includes this PR, you could clone the repo and run `npm run bundle` to build

The documentation for `npm install`: https://docs.npmjs.com/cli/install

Follow these steps:

1. `npm install https://github.com/riophae/vue-treeselect`
2. `cd node_modules/@riophae/vue-treeselect`
3. `npm install` install all dependencies for vue-treeselect
4. `npm run bundle` rebuild

Now you should be able to use the new version of vue-treeselect.

**Pull request merged**